test/new-e2e/examples/agentenv_updateenv_test.go

```diff
@@ -0,0 +1,41 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package examples
+
+import (
+	"testing"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/utils/e2e"
+	"github.com/DataDog/test-infra-definitions/components/datadog/agent"
+	ec2vm "github.com/DataDog/test-infra-definitions/scenarios/aws/vm/ec2VM"
+	"github.com/DataDog/test-infra-definitions/scenarios/aws/vm/os"
+	"github.com/stretchr/testify/assert"
+)
+
+type agentSuiteEx4 struct {
+	e2e.Suite[e2e.AgentEnv]
+}
+
+func TestVMSuiteEx4(t *testing.T) {
+	e2e.Run(t, &agentSuiteEx4{}, e2e.AgentStackDef(
+		[]e2e.Ec2VMOption{ec2vm.WithOS(os.UbuntuOS)},
+		agent.WithAgentConfig("log_level: debug"),
+	))
+}
+
+func (v *agentSuiteEx4) TestLogDebug() {
```

Comment:
This test works because `TestlogDebug` comes before `TestLogInfo` using lexicographic order.
What do you think about:
- Merging these two methods into a single one
- Removing `agent.WithAgentConfig("log_level: debug")` `from e2e.Run`
- Call twice `UpdateEnv` in the same test?
-